src/components/shared/AvatarUploader/AvatarUploader.tsx

```diff
@@ -34,115 +34,146 @@ const ACCEPTED_MIME_TYPES = {
   'image/png': [],
```

Comment:
@willm30  Its always been like this but since you are making changes, is there any reason not to allow uploading jpeg?